lib/fluent/plugin/out_s3.rb

```diff
@@ -272,6 +298,60 @@ def write(chunk)
                    else
                      @time_slice_with_tz.call(metadata.timekey)
                    end
+      
+      bucket_name = nil
+
+      if @s3_bucket =~ /\$\{.*\}/
+        @s3_bucket.scan(/\$\{([^\$\{\}]+)\}/) do |placeholder|
+          placeholder = placeholder.join
```

Comment:
```ruby
placeholders = placeholder.join
```

placeholder variable should contain multiple placeholder**s** information?
If so, this variable name should be plural form.